Dockerfile

```diff
@@ -6,7 +6,11 @@ RUN go mod download
 COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' \
         -o /bin/a.out ./cmd/cloudshell_open
+WORKDIR /tmp
+RUN wget https://github.com/buildpack/pack/releases/download/v0.2.1/pack-v0.2.1-linux.tgz && tar -xzf pack-v0.2.1-linux.tgz
 
 FROM gcr.io/cloudshell-images/cloudshell:latest
 RUN rm /google/devshell/bashrc.google.d/cloudshell_open.sh
+RUN apt update && apt install docker-ce -y
```

Comment:
ideally we shouldn't install `docker` in this image, as it should be present in the cloudshell base image already.
if it's not the right version or something like that, we should probably consider updating it. having two versions installed would probably cause an issue at some point later on.

Reply:
This should be resolved soon

Reply:
We do just need a new version and that can be done instead with `apt upgrade docker-ce`

Reply:
Let's get this fixed in cloudshell base image.
Here we're updating to latest version that we don't know whether is good or bad. At least we can follow cloudshell's release cycles.

Reply:
Looks like it has been. We'll just have to wait a little bit for it to get rolled out. I've removed the line.